trunk/SUAVE/Components/Energy/Networks/Battery_Propeller.py

```diff
@@ -256,12 +376,32 @@ def unpack_unknowns(self,segment):
             N/A
         """                  
         
+        # unpack the ones function
+        ones_row = segment.state.ones_row
+        
         # Here we are going to unpack the unknowns (Cp) provided for this network
         ss = segment.state
-        ss.conditions.propulsion.propeller_power_coefficient = ss.unknowns.propeller_power_coefficient
-        ss.conditions.propulsion.battery_voltage_under_load  = ss.unknowns.battery_voltage_under_load
-        
-        return
+        if segment.battery_discharge: 
+            ss.conditions.propulsion.propeller_power_coefficient = ss.unknowns.propeller_power_coefficient  
+        else:
+            ss.conditions.propulsion.propeller_power_coefficient = 0. * ones_row(1)
+            
+        if type(self.battery) == SUAVE.Components.Energy.Storages.Batteries.Constant_Mass.Lithium_Ion_LiFePO4_18650:
```

Comment:
again, we need to get these out of here in a better way

Reply:
addressed in prior comment